examples/main-xml/pom.xml

```diff
@@ -113,5 +113,21 @@
             </plugin>
         </plugins>
     </build>
-
+    <profiles>
+        <profile>
+            <id>camel.debug</id>
+            <activation>
+                <property>
+                    <name>camel.debug</name>
+                    <value>true</value>
```

Comment:
Why do you need an activation if your goal is to enable it with its id?

Reply:
to let the choice to users to use the profile id or the property. it gives more flexibility

Reply:
If so, I believe that you need to document it too as we cannot assume that the user will check the pom

Reply:
added